manifests/o/ojdkbuild/ojdkbuild/17.0020.8.1/ojdkbuild.ojdkbuild.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 17.0020.8.1
+PackageLocale: en-US
+Publisher: ojdkbuild open-source project
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: ojdkbuild OpenJDK JRE
```

Comment:
we should probably do ojdkbuild OpenJDK JRE 17?

Reply:
Yes.